tests/sharding/mainchain_handler/test_mainchain_handler.py

```diff
@@ -0,0 +1,63 @@
+from eth_tester.backends.pyevm.main import (
+    get_default_account_keys,
+)
+
+import eth_utils
+
+from evm.utils.address import generate_contract_address
+
+from tests.sharding.mainchain_handler.fixtures import (  # noqa: F401
+    mainchain_handler,
+)
+
+
+PASSPHRASE = '123'
+
+test_keys = get_default_account_keys()
+
+code = """
+num_test: public(num)
+
+@public
+def __init__():
+    self.num_test = 42
+
+@public
+def update_num_test(_num_test: num):
+    self.num_test = _num_test
+"""
+
+
+def test_tester_chain_handler(mainchain_handler):  # noqa: F811
```

Comment:
This test is titled `test_tester_chain_handler` but it's using the `mainchain_handler`.  Is that intentional?

Reply:
No it is not intentional. sorry I will rename it.